resolwe/process/tests/test_python_process.py

```diff
@@ -377,6 +377,37 @@ def test_get_collection(self):
         data = self.run_process("get-collection", {"collection_name": collection_name})
         self.assertEqual(data.output["collection_slug"], collection.slug)
 
+    @with_docker_executor
+    @tag_process("get-latest-process")
+    def test_get_process(self):
+        """Test process that gets process by slug.
+
+        The catch here is that multiple objects are returned, as there can be
+        multiple processes with the same slug but different versions. In such
+        case the object with the process with the latest version must be
+        returned.
+        """
+        process_slug = "multiple-versions"
+        self.assertFalse(Process.objects.filter(slug=process_slug).exists())
+
+        process1 = Process.objects.create(
+            persistence=Process.PERSISTENCE_TEMP,
+            version="1.0.0",
+            slug=process_slug,
+            contributor=self.admin,
+        )
+        data = self.run_process("get-latest-process", {"process_slug": process_slug})
+        self.assertEqual(data.output["process_pk"], process1.pk)
+
+        process2 = Process.objects.create(
+            contributor=self.admin,
+            persistence=Process.PERSISTENCE_TEMP,
+            version="1.0.1",
+            slug=process_slug,
+        )
+        data = self.run_process("get-latest-process", {"process_slug": process_slug})
+        self.assertEqual(data.output["process_pk"], process2.pk)
```

Comment:
This can pass even if the method doesn't work as intended. I think it would be much more robust if you bump the version of `process1` to `2.0.0` and rerun the process to ensure it returns a different result.